source/includes/kubernetes/_k8_namespaces.md

```diff
@@ -0,0 +1,165 @@
+        "selfLink": "/api/v1/namespaces/lab",
+        "uid": "74716eb1-a354-11e9-8c9b-1029384wedad"
+      },
+      "spec": {
+        "finalizers": [
+          "kubernetes"
+        ]
+      },
+      "status": {
+        "phase": "Active"
+      }
+    }
+  ],
+  "metadata": {
+    "recordCount": 3
+  }
+}
+```
+
+
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/namespaces</code>
+
+Retrieve a list of all namespaces in a given [environment](#administration-environments).
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/>*string* | The id of the namespace.  
+`metadata` <br/>*object* | The metadata of the the namespace
+`metadata.annotations` <br/>*map* | The annotations of the namespace
+`metadata.creationTimestamp` <br/>*string* | The date of creation of the namespace as a string
+`metadata.name` <br/>*string* | The name of the namespace
+`metadata.resourceVersion` <br/>*string* | An opaque value that represents the internal version of namespace object
+`metadata.selfLink` <br/>*string* | A URL representing the namespace object
+`metadata.uid` <br/>*object* | The UUID of the namespace
+`spec`<br/>*object* | The specification describes the attributes on a namespace.
+`spec.finalizers`<br/>*string* | An opaque list of values that must be empty to permanently remove object from storage
+`status`<br/>*object* | The status information of the namespace
+`status.phase`<br/>*string* | The status of the namespace. Possible statuses are Active, Terminating and Unknown
+
+
+
+<!-------------------- GET A NAMESPACE -------------------->
+
+### Get a namespace
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/k8s/anenvironment/namespaces/cert-manager"
+```
+> The above command returns JSON structured like this:
+
+```json
+{
+  "data": {
+    "id": "cert-manager",
+    "apiVersion": "v1",
+    "kind": "Namespace",
+    "metadata": {
+      "annotations": {
+        "kubectl.kubernetes.io/last-applied-configuration": "{\"apiVersion\":\"v1\",\"kind\":\"Namespace\",\"metadata\":{\"annotations\":{},\"name\":\"cert-manager\"}}\n"
+      },
+      "creationTimestamp": "2020-04-24T15:06:10.000-04:00",
+      "name": "cert-manager",
+      "resourceVersion": "110011696",
+      "selfLink": "/api/v1/namespaces/cert-manager",
+      "uid": "ab4376c2-bb6d-44ae-a576-f9b24f2ea624"
+    },
+    "spec": {
+      "finalizers": [
+        "kubernetes"
+      ]
+    },
+    "status": {
+      "phase": "Active"
+    }
+  }
+}
+```
+
+
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/namespaces/:id</code>
+
+Retrieve a namespace and all its info in a given [environment](#administration-environments).
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/>*string* | The id of the namespace.
```

Comment:
Same comments as above for the List Namespaces

Reply:
6b76e7d